### PR TITLE
Add codeowners file

### PR DESCRIPTION
**Pull request by breanatate**

### CODE REVIEW

**Review by bradoutlaw — CHANGES_REQUESTED** [review]
boutlaw -> bradoutlaw to resolve errors

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Health Connect
+/health-connect/ @abozowski @boutlaw @jessi3py
+
+#Health Platform
+/health-platform-v1/ @abozowski @boutlaw @jessi3py
+
+#Health Services on Wear OS
+/health-services/ @breanatate @yschimke
```
Comment (yschimke) [inline .github/CODEOWNERS:8]: I'm not sure I should be a CODEOWNER here.

I'm mainly helping out occasionally.

**Comment by yschimke** [comment]
Should this affect how we approach renovate dependency updates?

https://github.com/android/health-samples/pull/307/files

I've been following the Wear samples guides for updates, but maybe we want those split out if possible along these project lines?

**Comment by breanatate** [comment]
> Should this affect how we approach renovate dependency updates?
> 
> https://github.com/android/health-samples/pull/307/files
> 
> I've been following the Wear samples guides for updates, but maybe we want those split out if possible along these project lines?

Yes, I think this does affect that. I think the bulk of the HC/HP sample updates so far have been renovate-bot changes. And if we kept the CODEOWNERS file as is, I think it would mean that someone from HC would be required to also approve any renovate bot PRs.

**Comment by breanatate** [comment]
If I move renovate.json into the health-services folder does that solve things?

**Comment by yschimke** [comment]
> If I move renovate.json into the health-services folder does that solve things?

Let me check.

**Comment by breanatate** [comment]
> > If I move renovate.json into the health-services folder does that solve things?
> 
> Let me check. But no need to block this PR.

I somehow missed the second sentence 🤦🏾 

I think [includePaths](https://docs.renovatebot.com/configuration-options/#includepaths) does what we need

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Health Connect
+/health-connect/ @abozowski @boutlaw @jessi3py
```
[the remark this answers is not in the document]
Comment (breanatate) [reply .github/CODEOWNERS]: Sorry, I missed this! I'll send you a ping about getting added as a maintainer

**Comment by yschimke** [comment]
Well this is a better outcome

**Pull request merged**